db/offence_code_seeder_scheme_ten.rb

```diff
@@ -0,0 +1,29 @@
+  class OffenceCodeSeederSchemeTen
+    attr_reader :description, :class_letter, :contrary_to
+
+    def initialize(description, class_letter, contrary_to)
+      @description = description
+      @class_letter = class_letter
+      @contrary_to = contrary_to
+    end
+
+    def unique_code
+      modifier = 0
+      unique_code = code
+      unique_code = code(modifier += 1) while exists?(unique_code)
```

Comment:
💭  I lack context here (so bare with me on it 😅 ) but would it be better for the modifier to be a `SecureRandom` of some sort (most likely to be unique, could even be time based if required)?